ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -192,13 +198,14 @@ open class PeoplePicker: BadgeField {
         }
 
         personaListView.searchDirectoryState = .idle
-
+        delegate?.peoplePickerDidShowPersonaSuggestions?(self)
```

Comment:
at this point the people suggestion is actually shown? has the layout change?

Reply:
Thanks for pointing this out. I never had a deep understanding about update circle/setNeedsLayout/layoutIfNeeded. I just learned it. Added "layoutIfNeeded" after "setNeedsLayout" so that layoutSubviews would be called synchronously.